### PR TITLE
Alpha Help Tweaks

### PR DESCRIPTION
**Pull request by kingthorin**
pscanalpha.html > Updated headings to include scanner names (based on getName()), added missing entries, tweaked alpha sorting, and removed un-used entry.
Messages.properties > Removed un-used resource keys/messages.
ZapAddOn.xml > Incremented version and updated changes section.

### CODE REVIEW

**Comment by kingthorin** [comment]
@psiinon @thc202 if you're happy with how this is done I guess I could do the same for ascan alpha and include those tweaks in this PR as well.

File: src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html [hunk 1]
```diff
@@ -9,58 +9,68 @@
...
-<H2>HeartBleed</H2>
+<H2>HeartBleed (HeartBleedScanner)</H2>
 Passively scans for HTTP header responses that may indicate that the server is vulnerable to the critical HeartBleed OpenSSL vulnerability. 
 
-<H2>HTTP to HTTPS Insecure Transition in Form Post</H2>
+<H2>HTTP to HTTPS Insecure Transition in Form Post (InsecureFormPostScanner)</H2>
 This check looks for insecure HTTP pages that host HTTPS forms. The issue is that an insecure HTTP page can easily be hijacked through MITM and the secure HTTPS form can be replaced or spoofed.
 
-<H2>HTTPS to HTTP Insecure Transition in Form Post</H2>
+<H2>HTTPS to HTTP Insecure Transition in Form Load (InsecureFormLoadScanner)</H2>
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html]: The alert uses "Post".

**Comment by thc202** [comment]
I was expecting scanner names not class names, I think it's more useful the former for the user (e.g. to know which scanner disable or for alert filters), we don't show the classnames anywhere(?). Still, the classname is useful if you want to delve into the code, I'd add that as a note after the description though.

**Comment by kingthorin** [comment]
Ok I can switch it to be specifically what's returned by `getName()` I guess, but then we may as well just totally replace the headings?

**Comment by thc202** [comment]
Yeah, was wondering that, sounds good to me (like with "Strict Transport Security Header Scanner"?).

**Comment by kingthorin** [comment]
Exactly. Ok further update coming.

File: src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html [hunk 2]
```diff
@@ -96,14 +112,13 @@ <H2>Insecure Component</H2>
...
+Open redirects are one of the OWASP 2010 Top Ten vulnerabilities. This check looks at user-supplied input in query string parameters and POST data to identify where open redirects might be possible. Open redirects occur when an application allows user-supplied input (e.g. http://nottrusted.com) to control an offsite redirect. This is generally a pretty accurate way to find where 301 or 302 redirects could be exploited by spammers or phishing attacks.
 
 <H2>PII Disclosure scanner</H2>
 PII is information like credit card number, SSN etc. This check currently reports only numbers which match credit card numbers and pass Luhn checksum, which gives high confidence, that this is a credit card number.
 
-<H2>HTTP Server Response Header Scanner</H2>
-This checks response headers for the presence of a server header that contains version details.
-At LOW Threshold will raise an alert based on presence of the header field whether or not a version string is detected.
+<H2>Retrieved from Cache (RetrievedFromCacheScanner)</H2>
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html]: Remove the class name?

**Comment by thc202** [comment]
Thank you!

**Comment by kingthorin** [comment]
@psiinon look good?

**Pull request merged**